code/game/turfs/simulated.dm

```diff
@@ -53,6 +53,8 @@
 				wet_overlay = image('icons/effects/water.dmi', src, "ice_floor")
 			else
 				wet_overlay = image('icons/effects/water.dmi', src, "wet_static")
+		if(plane == PLANE_SPACE)
+			wet_overlay.plane = PLANE_SPACE_PARALLAX + 0.1
```

Comment:
```suggestion
		wet_overlay.plane = MID_TURF_LAYER
```

This makes it a lot less hacky, and forces it on all turfs just fine
![image](https://user-images.githubusercontent.com/25063394/89786351-b2de5c00-db13-11ea-86d1-40f92906c9a4.png)

Reply:
done